aten/src/ATen/native/native_functions.yaml

```diff
@@ -1306,6 +1306,18 @@
 - func: diff.out(Tensor self, int n=1, int dim=-1, Tensor? prepend=None, Tensor? append=None, *, Tensor(a!) out) -> Tensor(a!)
   variants: function
 
+- func: gradient.floatint(Tensor self, float? spacing=None, *, int? axis=None, int edge_order=1) -> Tensor[]
```

Comment:
Let's actually make everything after self kwarg-only for now. The *varargs of the NumPy version of this function worries me a little bit.